### DISCUSSION TITLE
Picgo插件问题

### DISCUSSION THREAD

**Discussion by Yafine** [Molunerfinn/PicGo#616] [Q&A]
插件一直显示安装中，这是怎么回事？应该怎么解决这个问题？

**Comment by Molunerfinn** [comment]
可以去picgo的日志所在目录，直接 `npm install picgo-plugin-xxx --save` （xxx为插件名）手动安装。如果发现这个步骤也很慢，说明是你的网络问题，最好挂个全局代理。

后续会支持配置npm的仓库源，用国内源会快一点

**Reply by Molunerfinn** [reply]
手动安装完，需要重启picgo才能生效

**Reply by Yafine** [reply]
> 续会支持配置npm的仓库源，用国内源会快一点

好的，我试下手动安装，谢谢

**Reply by Yafine** [reply]
> 手动安装完，需要重启picgo才能生效

![image](https://user-images.githubusercontent.com/31881437/105939165-9a4b0980-6093-11eb-84b1-d475e9f6116e.png)
报错了，看不懂这是什么意思？大佬求解

**Reply by Molunerfinn** [reply]
> > 手动安装完，需要重启picgo才能生效
> 
> ![image](https://user-images.githubusercontent.com/31881437/105939165-9a4b0980-6093-11eb-84b1-d475e9f6116e.png)
> 报错了，看不懂这是什么意思？大佬求解

我也看不懂，找你装的这个插件的仓库去提issue。
